docs/manual.md

```diff
@@ -583,6 +583,8 @@ http:
     allow-insecure: false
     no-follow-redirects: false # Setting this to true will NOT follow redirects
     timeout: 1000
+    request-header: # Optionally you can set request header values
```

Comment:
Should be `request-headers`, right?

Reply:
The change was merged already, someone submitted a documentation fix last week.

Next release will fix the header/request-headers inconsistency between the two features.